lib/utils/matrix_sdk_extensions/matrix_file_extension.dart

```diff
@@ -38,8 +38,8 @@ extension MatrixFileExtension on MatrixFile {
     return;
   }
 
-  void downloadImageInWeb(BuildContext context) {
-    Logs().d("MatrixFileExtension()::downloadImageInWeb()::download on Web");
+  void downloadFileInWeb(BuildContext context) {
```

Comment:
- download file and image are the same?

- Pls create a method handle `html.AnchorElement`.

Reply:
> download file and image are the same?
Since we download files from bytes, type doesn't matter IMO

> Pls create a method handle html.AnchorElement.
Why creating an other method for this part, imo it's overkill the method is readable. 
Could you give me a methods name for that ?